src/main/kotlin/team/themoment/gsmNetworking/domain/board/repository/BoardRepository.kt

```diff
@@ -1,6 +1,11 @@
 package team.themoment.gsmNetworking.domain.board.repository
 
 import org.springframework.data.jpa.repository.JpaRepository
+import org.springframework.data.jpa.repository.Query
 import team.themoment.gsmNetworking.domain.board.domain.Board
 
-interface BoardRepository : JpaRepository<Board, Long>, BoardCustomRepository
+interface BoardRepository : JpaRepository<Board, Long>, BoardCustomRepository {
+
+    @Query("select board from Board board where board.isPinned = true")
+    fun findPinnedBoards(): List<Board>
```

Comment:
```suggestion
    List<Board> findAllByPinnedIsTrue();
```

@Query 어노테이션을 사용하지 않고도 spring data jpa의 메서드로더 직관적으로 구현이 가능할것 같습니다. 
특별히 @Query 어노테이션으로 메서드를 작성하신 이유가 있으신가요?

Reply:
DML 생각하다가 @Query 어노테이션으로 구현을 했었는데

변경하였습니다.
[84013f7](https://github.com/themoment-team/GSM-Networking-server/pull/160/commits/84013f77117c2e233263f90b927764bb3c1f8a14)